.ci/integration.cloudbuild.yaml

```diff
@@ -806,8 +806,8 @@ steps:
           cassandra
 
   - id: "oracle"
-    name: golang:1
-    waitFor: ["compile-test-binary"]
+    name: ghcr.io/oracle/oraclelinux8-instantclient:21
```

Comment:
Can we continue to test with coverage? We could possiblyinstall these dependencies in a different step than the test runner